CTN-001.tex

```diff
@@ -107,6 +111,33 @@ \section{Introduction}
   \label{fig:focal_plane_3}
 \end{figure}
 
+
+\begin{figure}
+  \centering
+  \includegraphics[width=\textwidth]{figures/ComCam_focal_plane_CTN_001_FIG1.pdf}
+  \caption{ComCam focal plane layout. Code source: \url{https://github.com/lsst/ctn-001/code/}}
+  \label{fig:comcam_focal_plane_1}
+\end{figure}
+
+\clearpage
+
+\begin{figure}
+  \centering
+  \includegraphics[width=\textwidth]{figures/ComCam_focal_plane_CTN_001_FIG2.pdf}
+	\caption{ComCam focal plane layout including raft and CCD serial numbers (see text). The raft and CCD serial numbers come from the \emph{eTraveler} database accessed using {\tt{datacat-utilities}} (\url{https://github.com/lsst-camera-dh/datacat-utilities}).}
+  \label{fig:comcam_focal_plane_2}
+\end{figure}
+
+\clearpage
+
+\begin{figure}
+  \centering
+  \includegraphics[width=\textwidth]{figures/comcam_photo.jpg}
+  \caption{In one of the early images taken with Rubin's 144-megapixel test camera, Rubin staff gathered for a group photo inside the observatory dome, with the Simonyi Survey Telescope pointed at horizon. This "pinhole" image was created using the camera's sensors but without using lenses or mirrors to focus light. Instead of forming a detailed, zoomed-in picture, this technique gives a soft, unfocused image. The visible grid pattern of the camera's sensor array overlays the image. Source: \url{https://rubinobservatory.org/gallery/}}
```

Comment:
If the caption is also from the gallery, I'd recommend changing "Source" to "Image and caption source"